egs/csj/s5/run.sh

```diff
@@ -277,6 +283,9 @@ wait
 # Karel's DNN recipe on top of fMLLR features
 local/nnet/run_dnn.sh
```

Comment:
can you please move all the MMI training to something like local/run_mmi.sh and have it commented out in the top-level script, and also comment out local/nnet/run_dnn.sh?  (Once you add the speed perturbing, run_tdnn_chain.sh will be comfortably better than that result).